### DISCUSSION TITLE
How do I migrate from Shovel to Scoop ?

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [Ash258/scoop#308] [Q&A] [answered]
```
PS E:\> scoop --version
PowerShell version: 7.4.2
Useragent: Shovel/1.0 (+https://shovel.ash258.com) PowerShell/7.4 (Windows NT 10.0.19045.0; Win64; x64;)
Current Scoop (Shovel) version:
52fe1646 (HEAD -> main, origin/master, origin/main, origin/HEAD) fix(Diagnostic): Better handling of migrated repos
```

I'd like to migrate back to https://github.com/ScoopInstaller/Install#scoop-uninstaller, I already have a lot of shovel installed apps. How do I go about that without breaking everything?

**Comment by Ash258** [comment, marked as answer]
Cannot tell.

It should be enough to rename `scoop-install.json` and `scoop-manifest.json` to version without `scoop-` prefix.

Then clone the original scoop to scoop\current
